upstairs/src/upstairs.rs

```diff
@@ -2109,22 +2093,18 @@ pub(crate) mod test {
         upstairs.force_active().unwrap();
 
         // op 0
-        upstairs
```

Comment:
Did `cargo fmt` do this?
Maybe it decided to change things because the `.await;` went away?

Reply:
Yup, thanks `cargo fmt`